compiler/test/compilable/test23490.d

```diff
@@ -0,0 +1,4 @@
+// https://issues.dlang.org/show_bug.cgi?id=23490
+// PERMUTE_ARGS:
```

Comment:
Does this have a use?

Reply:
Maybe not - I forget whether compilable tests permute all default compiler flags.